backend/firebaseFunctions.js

```diff
@@ -70,28 +70,76 @@ export async function createCircle(data) {
 	data.admin[`${UID}`] = true;
 	data.owner = UID;
 
+	let circles = Object.keys((await readData(`prayer_circle/circles`)) || {});
+	const allAdminCodes = new Array(circles.length);
```

Comment:
I think the code here can be simplified a lot. Start with only making one array, `codes`, as there is no real reason for having 2. If the codes were formatted differently, then maybe, but because we aren't then it doesn't matter.